public/img/icons/arrow-back.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
let's make the icons react components and store them inside src/components/icons